src/components/HowToPlay/HowToPlayDialog.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import {
+  Dialog,
+  DialogActions,
+  DialogButton,
+  DialogContent,
+  DialogTitle,
+} from '@rmwc/dialog';
+import styles from './HowToPlayDialog.module.css';
+import { ArrowButtons } from '../ArrowButtons/ArrowButtons';
+import { useEffect } from 'react';
+import { HowToPlayContent } from './HowToPlayContent';
+import { HowToPlayBoardState } from '../../state/AppState';
+import { Button } from '@rmwc/button';
+import { ArrowRightIcon } from '../Icons';
+
+interface ActionBarProps {
+  onClose: () => void;
+  onForward: () => void;
+  onBackward: () => void;
+  stepNumber: number;
+  maxStepNumber: number;
+}
+
+function ActionBar(props: ActionBarProps) {
+  const { onClose, onForward, onBackward, stepNumber, maxStepNumber } = props;
+
+  const isLastStep = stepNumber === maxStepNumber;
+  const isFirstStep = stepNumber === 0;
+
+  return (
+    <>
+      {isFirstStep && <DialogButton action="close">Cancel</DialogButton>}
+      <ArrowButtons
+        value={stepNumber}
+        maxValue={maxStepNumber}
+        minValue={0}
+        onInteraction={(forward) => {
+          if (forward) onForward();
+          else onBackward();
+        }}
+        handleKeyboard={true}
+        leftButtonConfig={{ buttonProps: { raised: false }, hide: isFirstStep }}
+        rightButtonConfig={{
+          buttonProps: { raised: false },
+          hide: isLastStep,
+        }}
+      />
+      {isLastStep && (
+        <Button
+          raised={true}
+          trailingIcon={{ icon: <ArrowRightIcon />, 'aria-hidden': true }}
+          onClick={onClose}
+        >
+          Let's Play
+        </Button>
+      )}
+    </>
+  );
+}
+
+export interface HowToPlayDialogProps {
+  onOpen: () => void;
```

Comment:
We track the open state of the dialog in the state and react according to it. Currently:
- open -> reset state, start dispatching state actions (saga)
- close -> cancel sending more state actions

But you're right we actually wouldn't need:
1. tracking the open state
2. start dispatching state change actions on open because nothing is happening on the first step
3. we can move the reset of the state to the close action

I would opt for removing 1-3. What do you think?

Reply:
Done